src/screens/OrganizationEvents/OrganizationEvents.tsx

```
@@ -80,11 +80,35 @@ import {
+      edges?: Array<{
+        node?: {
+          id?: string;
+          name: string;
+        };
+      }>;
+    };
+  };
+}
+
 const timeToDayJs = (time: string): Dayjs => {
   const dateTimeString = dayjs().format('YYYY-MM-DD') + ' ' + time;
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)